storage/file_test.go

```diff
@@ -1,6 +1,7 @@
 package storage
 
 import (
+	"azure-sdk-for-go/storage"
```

Comment:
Ahhh thanks!

Out of interest, how did you figure that out? Looking at the Travis output I didn't see it mentioned/hinted anywhere. Did you just read the code and see the issue or should I be looking somewhere else for clues?

Thanks

Ken